### PR TITLE
VR Worlds (Workshop Bounty)

### PR DESCRIPTION
**Pull request by wollygfx**

### CODE REVIEW

**Comment by wollygfx** [comment]
Demo video: https://hackclub.slack.com/files/U015HRC40H4/F01F2DD3YBC/workshop.mov

**Review by sampoder — CHANGES_REQUESTED** [review]
This is so cool @wollygfx!!!!!!!

It's simple but you make something EPIC. I love it! 

Just a tiny few things that need fixing before approving:

- Could you check that all the assets are available? Some are missing.
- Could you clean up the Markdown a bit, not the biggest deal tho.
- Could you use a langagues link for repl.it, makes it easier for club members
- Remember to use html throughout for markdown syntax highlighting
- For the expansion demos could you add the source code
- Check the grammar, just general check ups nothing big. Grammarly could probs catch them all

I think this is so so close and I'm so so excited! Gotta close this but soon soon I promise :)))

Looking forward,

Sam

**Comment by wollygfx** [comment]
Hey @sampoder 
Here's my current progress: 
- [x] Could you check that all the assets are available? Some are missing.
- [x] Could you clean up the Markdown a bit, not the biggest deal tho.
- [ ] Could you use a langagues link for repl.it, makes it easier for club members
- [ ] Remember to use html throughout for markdown syntax highlighting
- [ ] For the expansion demos could you add the source code
- [x] Check the grammar, just general check ups nothing big. Grammarly could probs catch them all
There are some things that I don't understand yet though:

1. What do you mean by this?

> Could you use a langagues link for repl.it, makes it easier for club members

2. Also, this:

> Remember to use html throughout for markdown syntax highlighting

and finally, the demo links are made by other people, i might not be able to find their source codes... Is okay if i just leave it like that?

**Comment by sampoder** [comment]
1. I mean like using repl.it/languages/html as the link

2. i mean making your code blocks like:

````
\```html
html
\```
````

3. @MatthewStanciu thoughts?

**Comment by sampoder** [comment]
ignore the black slashes

**Comment by wollygfx** [comment]
@sampoder Wait but 
1. That link is already there
2. All the code blocks are already like that too

File: workshops/VR World/README.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+## Set-up
+This workshop requires a very basic knowledge of HTML. Don’t worry if you get stuck at some point in the workshop, everything is explained the best way for you to understand!
+
+For this workshop we will use [Repl.it](https://repl.it/), a free, online code editor. Click [here](https://repl.it/languages/html) to get started with a new HTML project on repl.it.
+
+![Setup](https://cloud-qbmylslty.vercel.app/0image.png)
+
+##  Setting up A-frame
+To get started, we'll need to link A-frame with our HTML document, there are multiple ways to do this that you can find [here](https://aframe.io/docs/1.0.0/introduction/installation.html), we'll just need to drop a `<script>` tag pointing to the CDN build:
+```js
```
Comment (sampoder) [inline workshops/VR World/README.md:21]: not here ;)

**Comment by sampoder** [comment]
Oh yep missed that, open up another PR and I'll review

File: workshops/VR World/README.md (hunk 1 above)
Comment (wollygfx) [reply workshops/VR World/README.md:21]: the <script> tag?

Comment (sampoder) [reply workshops/VR World/README.md:21]: the ```js should be ```html

Comment (sampoder) [reply workshops/VR World/README.md:21]: omg \``` js should be \```html

Comment (wollygfx) [reply workshops/VR World/README.md:21]: Oh yeah, i'll change that

**Pull request closed without merging**